shop_manager_design.md

```diff
@@ -0,0 +1,315 @@
+# {{TABLE NAME}} Model and Repository Classes Design Recipe
+
+_Copy this recipe template to design and implement Model and Repository classes for a database table._
+
+## 1. Design and create the Table
+
+As a shop manager
+So I can know which items I have in stock
+I want to keep a list of my shop items with their name and unit price.
+
+As a shop manager
+So I can know which items I have in stock
+I want to know which quantity (a number) I have for each item.
+
+As a shop manager
+So I can manage items
+I want to be able to create a new item.
+
+As a shop manager
+So I can know which orders were made
+I want to keep a list of orders with their customer name.
+
+As a shop manager
+So I can know which orders were made
+I want to assign each order to their corresponding item.
+
+As a shop manager
+So I can know which orders were made
+I want to know on which date an order was placed. 
+
+As a shop manager
+So I can manage orders
+I want to be able to create a new order.
+
+
+```
+# EXAMPLE
+
+Table: stock
+
+Columns:
+id | name | price | quantity
+
+Table: orders
+
+Columns: 
+id | cname | time | date
+
+```
+
+## 2. Create Test SQL seeds
+
+Your tests will depend on data stored in PostgreSQL to run.
+
+If seed data is provided (or you already created it), you can skip this step.
+
+```sql
+
+CREATE TABLE stocks( 
+id SERIAL PRIMARY KEY,
+name text,
+price int,
+quantity int
+);
+
+CREATE TABLE orders(
+id SERIAL PRIMARY KEY,
+cname text,
+time text,
+date text,
+stock_id int,
+constraint fk_stock foreign key(stock_id) references 
+stocks(id)
+);
+
+INSERT INTO "public"."stocks" ("id", "name", "price", "quantity" ) VALUES 
+(1, 'Item1', 1, 1),
```

Comment:
perhaps price here could be some kind of float, think about how we could do this.